.travis.yml

```diff
@@ -7,7 +7,7 @@ services:
   - postgresql
 
 php:
-  - 7.3
+  - 7.2
```

Comment:
Without the 7.4 in here too you lose the matrix. I really just pointed out to remove the middle line 7.3.

Reply:
Again: min+max

Reply:
Sorry. Yes makes sense.